### ISSUE TITLE
Citrix netscalar details

### ISSUE THREAD

**Issue by towster** [librenms/librenms#6210]
DO NOT DELETE THIS INFORMATION.

> Please read this information carefully.

GitHub issues is for  bugs, please do not post issues asking for help or how to do X, Y or Z. 
You can use our irc channel ##librenms on freenode to ask questions or our [community site](https://community.librenms.org).
If you have a feature request please post this on our [community site](https://community.librenms.org/c/feature-requests).

Please confirm each of the sections below by putting an `x` in the box like `[x]`.

- [x ] Is your install up to date? [Updating your install](http://docs.librenms.org/General/Updating/)
      Please do not submit an issue if your install is not up to date within the last 24 hours or on a stable monthly release.
- [x ] Please include all of the information between the `====================================` section of `./validate.php` which you can run from the cli.
- [x ] Please provide ALL info asked for [here](http://docs.librenms.org/Support/FAQ/#faq20).
- [x ] Please provide as much detail as possible.
- [x ] Please do NOT post more than 10 lines of debug information here, use a pastebin service or GitHub Gists.

I noticed that the netscalar is identified as a loadbalancer but there are no details for it like there is for the F5. Here is details for one of the netscalar devices. There may not be much info since they have not been fully configured yet. 

Here is the discovery, poller, bulkwalk
https://p.libren.ms/view/017c65af
https://p.libren.ms/view/03714372
https://p.libren.ms/view/cd6b8e51

Here is the MIB file
[NS-MIB-smiv2.mib.txt](https://github.com/librenms/librenms/files/848695/NS-MIB-smiv2.mib.txt)

**Comment by laf** [comment]
Turn the netscaler polling module on. If that works let us know. If it doesn't provide you the data you need pls actually provide info on what you want to see.

**Comment by towster** [comment]
Note that prior to doing anything both the F5 and the netscalar appeared under the loadbalancer device type. Also the data under F5 was already being collected without any poller changes.

I added to config.php the following:
```
$config['poller_modules']['netscaler-vsvr']              = 1;
$config['poller_modules']['loadbalancers']               = 1;
```
I re-ran discovery of the devices and waited for polling to run a couple of times. I did not see anything. I went to edit - modules and saw that netscalar was still disabled. 
![capture](https://cloud.githubusercontent.com/assets/1965956/24055910/ea98dabc-0b0f-11e7-9a71-0c7143efac02.PNG)
I went ahead and enabled netscalar but I dont understand why it was disabled via the OS even after enabling globally. Then I re-ran discovery and waited for polling again. Again I did not see anything right off. I started looking harder and the only thing I could found was this:
![capture](https://cloud.githubusercontent.com/assets/1965956/24056041/5eafb1c8-0b10-11e7-8101-960ecefadaea.PNG)
Now it is possible that more data will appear once the netscalar is fully configured and in use. I am hoping that it collects more than just those metrics. Regardless though the look and feel seems different. Here are some examples of what I mean.

Loadbalancer list - No platform or OS version for citrix
![image](https://cloud.githubusercontent.com/assets/1965956/24056298/25093592-0b11-11e7-9407-6f4ab5815895.png)

Viewing an F5 device has a load balancer tab
![image](https://cloud.githubusercontent.com/assets/1965956/24056381/683ff288-0b11-11e7-8143-014053d3fc62.png)

From within the loadbalancer tab it lists out pools, members and graphs things like connections, bytes and packets. I can provide screenshots of those if desired.
